aopy/visualization.py

```diff
@@ -1,2 +1,165 @@
 # visualization.py
-# code for general neural data plotting (raster plots, multi-channel field potential plots, psth, etc.)
\ No newline at end of file
+# code for general neural data plotting (raster plots, multi-channel field potential plots, psth, etc.)
+
+import matplotlib
+import matplotlib.pyplot as plt
+from scipy.interpolate import griddata
+from scipy.interpolate.interpnd import _ndim_coords_from_arrays
+from scipy.spatial import cKDTree
+import numpy as np
+import os
+
+def savefig(base_dir, filename, **kwargs):
```

Comment:
We can add more functionalities to save_fig such as ability to save as a pdf. 
 
After some thoughts, I think it is useful to keep this function for saving. And add another function for figure settings. What do you think?